<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddScientistDialog</class>
 <widget class="QDialog" name="AddScientistDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="radioButton_if_female">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_if_male">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add_scientist">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add scientist to list</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_add_scientist_window">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_scientist_name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_scientist_year_of_birth">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_scientist_year_of_death">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_scientist_photo">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>88</width>
     <height>177</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_scientist_name">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_scientist_year_of_birth">
          <property name="text">
           <string>Year of birth:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_scientist_year_of_death">
          <property name="text">
           <string>Year of death:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_scientist_gender">
          <property name="text">
           <string>Gender:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_add_scientist_photo">
        <property name="text">
         <string>Add photo:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_add_scientist_info">
        <property name="text">
         <string>Add info:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>radioButton_if_female</zorder>
  <zorder>radioButton_if_male</zorder>
  <zorder>pushButton_add_scientist</zorder>
  <zorder>cancel_add_scientist_window</zorder>
  <zorder>input_scientist_name</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>input_scientist_year_of_birth</zorder>
  <zorder>input_scientist_year_of_death</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>label_scientist_photo</zorder>
 </widget>
 <tabstops>
  <tabstop>input_scientist_name</tabstop>
  <tabstop>input_scientist_year_of_birth</tabstop>
  <tabstop>input_scientist_year_of_death</tabstop>
  <tabstop>radioButton_if_female</tabstop>
  <tabstop>radioButton_if_male</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>pushButton_add_scientist</tabstop>
  <tabstop>cancel_add_scientist_window</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
